examples/morpheusvm/throughput/helper.go

```diff
@@ -61,3 +74,13 @@ func (*SpamHelper) GetTransfer(address codec.Address, amount uint64, memo []byte
 		Memo:  memo,
 	}}
 }
```

Comment:
Passing comment... but this is so weird that this is on this type since it's entirely unrelated to the struct